watcher/watcher.go

```diff
@@ -530,7 +530,12 @@ func (wh *WatchHandler) getParentWorkloadForPod(pod *core1.Pod) (workloadinterfa
 	if err != nil {
 		return nil, err
 	}
+
 	kind, name, err := wh.k8sAPI.CalculateWorkloadParentRecursive(wl)
+	if kind == "Node" {
```

Comment:
I have not fixed changed `CalculateWorkloadParentRecursive` since the Node is indeed the parent, and the function does what is supposed to do. We just have a specific use-case of vulnerability scanning, where we don't consider the Node a parent.